state/errors.go

```diff
@@ -51,6 +51,10 @@ type (
 	ErrNoABCIResponsesForHeight struct {
 		Height int64
 	}
+
+	ErrCannotLoadState struct {
```

Comment:
note to reviewers: this PR adds this error to state, because evidence package returns it at some point: https://github.com/cometbft/cometbft/pull/1284/files#diff-4f5e7dc5896be906b72cb46fb61a2f935a428af838090279e7a4bded0679818dR58

Reply:
Makes sense to me